docs/Oscilloscope.md

```diff
@@ -0,0 +1,54 @@
+# How to use Oscilloscope
+
+## Table of Content
+1. Introduction
+2. Layout
+3. Features and Components
+    - Channel Parameters
+    - Timebase and Trigger
+    - Data analysis
+    - XY Plot
+4. How to Use
+5. Video demo    
+
+## Introduction
+Pocket Science Lab contains an Oscilloscpe which has lot of functions which are available in a commercial grade Oscilloscope. The PSLab android application communicates with the Pocket Science Lab and gives the user a real experience on using a commercial Oscilloscope. This document will help a user on available functionalities, how to use Pocket Science Lab with PSLab android application and some demo videos.  
+
+## Layout
+
+<table>
+    <tr>
+        <td><img src="/docs/screenshots/oscilloscope_channel_params.png"></td>
+        <td><img src="/docs/screenshots/oscilloscope_data_analysis.png"></td>
+    </tr>
+</table>
+<table>
+    <tr>
+        <td><img src="/docs/screenshots/oscilloscope_timebase_and_trigger.png"></td>
+        <td><img src="/docs/screenshots/oscilloscope_xy_plot.png"></td>
```

Comment:
@Vikum94  Why have you put this image of app crashing?

Reply:
That was a mistake. Now it's updated :+1: